### PR TITLE
Add growth monitoring report

### PR DESCRIPTION
**Pull request by mkangia**
## Summary

Adding more reports after https://github.com/dimagi/commcare-hq/pull/29056/files
This PR,
Adds a report for growth monitoring.

## Feature Flag

`USER_CONFIGURABLE_REPORTS`

## Safety Assurance

- [x] Risk label is set correctly
- [x] All migrations are backwards compatible and won't block deploy
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [x] If QA is part of the safety story, the "Awaiting QA" label is used
- [x] I have confidence that this PR will not introduce a regression for the reasons below

### Safety story

This PR just adds a new data source and report for the project. It is not really live so there should be no scale implications of this as well.

### Rollback instructions

Ideally this should not be rolled back because apps would be using it.

- [ ] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: custom/nutrition_project/ucr/reports/growth_monitoring_v1.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        "type": "choice_list",
+        "field": "gender",
+        "datatype": "string",
+        "choices": [
+          {
+            "display": "Male",
+            "value": "M"
+          },
+          {
+            "display": "Female",
```
Comment (Rohit25negi) [inline custom/nutrition_project/ucr/reports/growth_monitoring_v1.json:72]: Do you think more adding more indexes with these columns would be beneficial here seeing there are more filters in this report.

**Review by Rohit25negi — COMMENTED** [review]
left one question about indexes

File: custom/nutrition_project/ucr/reports/growth_monitoring_v1.json (hunk 1 above)
Comment (mkangia) [reply custom/nutrition_project/ucr/reports/growth_monitoring_v1.json]: thanks for flagging that @Rohit25negi . I cross checked that gender and completed time are not being used in the app, so i have removed them. The only one is dob which is used to check for age, so the checks are like dob - today = certain number of days, I guess an index won't help there.

Comment (Rohit25negi) [reply custom/nutrition_project/ucr/reports/growth_monitoring_v1.json]: offlined. Looks good

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**